packages/dbadmin/src/tools/init-schema-with-history.ts

```diff
@@ -362,17 +362,9 @@ export function initSchemaWithHistory(schemaName: string): string {
             AS
         $$ 
         DECLARE repo_version integer; BEGIN
-                SELECT value INTO repo_version FROM ${CURRENT_DATA} WHERE key = '${CURRENT_DATA_REPO_VERSION_KEY}';
-            IF NOT EXISTS (SELECT FROM ${PROPERTIES_TABLE_HISTORY} 
-                            WHERE property = NEW.property AND node_id = NEW.node_id 
-                          )
-            THEN 
-                INSERT INTO ${PROPERTIES_TABLE_HISTORY} 
+            SELECT value INTO repo_version FROM ${CURRENT_DATA} WHERE key = '${CURRENT_DATA_REPO_VERSION_KEY}';
```

Comment:
I would consider adding some comments as the SQL is not super obvious. To me it is not immediately obvious why in some parts we want always to do an insert while in other portions we want to differentiate depending on something existing or not

Reply:
I can add some explanation.
The difference between insert (new) and update (existing) is done in the server, not in the SQL.
I did remove all distinctions (for properties, containments, references) in the SQL for this reeason.

Reply:
Added some comments, hopefully making things more clear.